test/acceptance/environments/local.yml

```diff
@@ -1,7 +1,19 @@
 # HTTP settings
-url: https://httpbin.org
+http_bin_url: https://httpbin.org
```

Comment:
we can reorganize as config:
```
httpbin:
    url: https://httpbin.org
```
And use it as `[CONF:httpbin.url]`

Reply:
Done